tests/integration/test_store.py

```diff
@@ -725,6 +725,7 @@ def send_buckets(buckets):
     produced_buckets.sort(key=lambda b: (b["name"], b["value"]))
     for bucket in produced_buckets:
         del bucket["timestamp"]
+        del bucket["received_at"]
```

Comment:
This test was very tricky to update, we might want to follow up with a PR to fix timestamps across all of our tests.

Reply:
Can we not do this better now?